users_app/pipeline/pipeline.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth.models import Group
+
+from users_app import ALLOWED_AP_ACCOUNTS
+
+
+def add_user_to_group(is_new, user, *args, **kwargs):
+    if is_new and user.email in ALLOWED_AP_ACCOUNTS:
+        ap_admin_group = Group.objects.get(name='ap_admin')
+        ap_manager_group = Group.objects.get(name='ap_manager')
+        user.groups.add(ap_admin_group, ap_manager_group)
+
+    elif is_new and user.email.endswith('@eventbrite.com'):
+        ap_reporter_group = Group.objects.get(name='ap_reporter')
+        user.groups.add(ap_reporter_group)
+
+    elif is_new:
+        supplier_group = Group.objects.get(name='supplier')
+        user.groups.add(supplier_group)
+
+    return {
+            'is_new': is_new,
+            'user': user
+        }
```

Comment:
The condition is different but the result is always the same (they are added to the group). It is right? What is the following when they are already in the group?

Reply:
-There is different groups: depending on the condition, the AP is added to a group.
-If they are in a group (is_new=False) I think there's no need to re-assign them.